service/src/main/java/org/mskcc/cmo/metadb/service/impl/MessageHandlingServiceImpl.java

```diff
@@ -18,6 +18,8 @@
 import org.mskcc.cmo.metadb.model.MetadbSample;
```

Comment:
considering refactoring this message handling service to remain igo-specific and introduce a new message handler that is specific to handling DMP/clinical samples and updates